### ISSUE TITLE
svonedsolver.exe doesn't exist on windows

### ISSUE THREAD

**Issue by Noob0211** [SimVascular/SimVascular#995]
**Describe the bug**
Following the tutorial of ROM Simulation, I finally failed with the warning that the executable 1D solver didn't exist in the folder where I installed it.
I have installed the latest Simvascular and svOneDSolver in my windows, but in the svOneDSolver folder, I just find 2 files: "License.rtf", "release-date", actually the "svonedsolver.exe" does not exist in the installing folder.

**Screenshots**
![exe not exist](https://user-images.githubusercontent.com/34793082/193877280-c4fc842e-1300-40bb-bc29-6195f2d4f046.png)

**Desktop:**
 - OS: Windows 11
 - SimVascular Version 2022-07-22
 - svOneDSolver Version 2022-07-22
 - Medical Image Filetype DICOM(Demo File)

**Additional context**
I'm really confused about this, and I appreciate it if there is someone who can help me on this!

**Comment by Noob0211** [comment]
I didn't find "svonedsolver.exe" in the svonedsolver-installing folder.
![svonedsolver folder](https://user-images.githubusercontent.com/34793082/193881491-7b13b1eb-cc1e-4443-a3d6-425f4e111378.png)
But in the Simvascular-installing folder, I find the executable 1D solver.
![simvascular folder](https://user-images.githubusercontent.com/34793082/193881658-acc1eaf1-a7ea-459e-b7f4-c2a4cdf17a40.png)

Could you please give me some advice on how to fix this problem?
Thanks a lot!

**Comment by Noob0211** [comment]
Thanks for your help. I have tried uninstalling the svOneDSolver through the program "svOneDSolver-Windows-2022-07.msi", after which I still found the file "svOneDSolver.exe" existing in the SimVascular install directory. Then it may not be the cause that the svOneDSolver installer put svOneDSolver.exe in the SimVascular install directory, it seems that there may exist some other problem with the svOneDSolver installer, for example, it doesn't contain the executable solver file? I guess. 
![after_uninstall](https://user-images.githubusercontent.com/34793082/193953603-e4e06664-c7fb-415c-b665-59ef8d3b85f3.png)
![after_uninstall_reference](https://user-images.githubusercontent.com/34793082/193953626-13ae4540-3bff-413c-b189-098a3b45339d.png)
Then, I went to the preference pannel, and after specifying the path of "svOneDSolver.exe" in the simvascular install directory to the "Path to 1D Solver executable", finally I can run 1D simulation!
Thanks again!

![specify_path](https://user-images.githubusercontent.com/34793082/193954554-e7583359-ab77-4c0e-94fc-cf26defaf71b.png)

![complete](https://user-images.githubusercontent.com/34793082/193954582-2929a4d7-158c-465f-bbf8-1258aa4c5abe.png)
